packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/guides/Migration.tool.md

```diff
@@ -0,0 +1,522 @@
+# Terra Functional Testing
```

Comment:
Somewhere i'd specify that this guide is for migrating from terra-toolkit to terra-functional-testing v1 since we will have a terra-functional-testing v2 in the future.

Reply:
I added a note to the first paragraph summary:

https://github.com/cerner/terra-toolkit/pull/574/commits/42750ce4526f2f3d447820f56d3dd5b0f3b8bfc2